### ISSUE TITLE
Authr-ClientPin1-GetKeyAgreement P-1 : alg is incorrect

### ISSUE THREAD

**Issue by eugenIFX** [fido-alliance/conformance-tools-issues#164]
## What protocol and version of the protocol are you testing?

FIDO2

## What is your implementation class?

Authr

## What is the version of the tool are you using?
0.9.413

## Issue description

In the spec is saying that the keyAgreement.alg is ECDH P-256 (-7).

**Comment by herrjemand** [comment]
Resolved in v0.9.417

**Comment by herrjemand** [comment]
@eugenIFX Actually is was suppose to be -25

Here is [CTAP specs](https://drafts.fidoalliance.org/fido-2/latest/fido-client-to-authenticator-protocol-v2.0-wd-20180426.html#authenticator-power-up-configuration):
> Generate an ECDH P-256 key pair called "authenticatorKeyAgreementKey" denoted by (a, aG) where "a" denotes the private key and "aG" denotes the public key.

In COSE the example for ECDH says -25 https://tools.ietf.org/html/draft-ietf-cose-msg-24#appendix-C.3.1:
```
[
       / protected / h'a10101' / {
           \ alg \ 1:1 \ AES-GCM 128 \
         } / ,
       / unprotected / {
         / iv / 5:h'c9cf4df2fe6c632bf7886413'
       },
       / ciphertext / h'7adbe2709ca818fb415f1e5df66f4e1a51053ba6d65a1a0
   c52a357da7a644b8070a151b0',
       / recipients / [
         [
           / protected / h'a1013818' / {
               \ alg \ 1:-25 \ ECDH-ES + HKDF-256 \
             } / ,
           / unprotected / {
             / ephemeral / -1:{
               / kty / 1:2,
               / crv / -1:1,
               / x / -2:h'98f50a4ff6c05861c8860d13a638ea56c3f5ad7590bbf
   bf054e1c7b4d91d6280',
               / y / -3:true
             },
             / kid / 4:'meriadoc.brandybuck@buckland.example'
           },
           / ciphertext / h''
         ]
       ]
     ]
```

**Comment by herrjemand** [comment]
Need to clarify with TWG

**Comment by herrjemand** [comment]
General consensus in implementations in -25. So changing to -25

**Comment by herrjemand** [comment]
Resolved in v0.9.420
